client/rest-high-level/src/test/java/org/elasticsearch/client/DataFrameRequestConvertersTests.java

```diff
@@ -66,14 +68,17 @@ public void testPutDataFrameTransform() throws IOException {
         PutDataFrameTransformRequest putRequest = new PutDataFrameTransformRequest(
                 DataFrameTransformConfigTests.randomDataFrameTransformConfig());
         Request request = DataFrameRequestConverters.putDataFrameTransform(putRequest);
-
+        assertThat(request.getParameters(), not(hasKey("defer_validation")));
         assertEquals(HttpPut.METHOD_NAME, request.getMethod());
```

Comment:
While you are, could you change "assertEquals" to "assertThat(..., equalTo(...)) so that the usage of assertions is consistent in the whole method.